src/Commands/InstallCommand.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Spatie\LaravelPackageTools\Commands;
+
+use Illuminate\Console\Command;
+use Spatie\LaravelPackageTools\Package;
+
+class InstallCommand extends Command
+{
+    protected Package $package;
+
+    protected bool $shouldPublishConfigFile = false;
+
+    protected bool $shouldPublishMigrations = false;
+
+    protected ?string $copyServiceProviderInApp = null;
+
+    protected ?string $starRepo = null;
+
+    public function __construct(Package $package)
+    {
+        $this->signature = $package->shortName() . ':install';
+
+        $this->package = $package;
+
+        parent::__construct();
+    }
+
+    public function handle()
+    {
+        if ($this->shouldPublishConfigFile) {
+            $this->info('Publishing config file...');
+
+            $this->callSilently("vendor:publish", [
+                '--tag' => "{$this->package->shortName()}-config",
+            ]);
+        }
+
+        if ($this->shouldPublishMigrations) {
+            $this->info('Publishing migration...');
+
+            $this->callSilently("vendor:publish", [
+                '--tag' => "{$this->package->shortName()}-migrations",
+            ]);
+        }
+
+        if ($this->starRepo) {
+            if ($this->confirm('Would you like to star our repo on GitHub?')) {
+                $repoUrl = "https://github.com/{$this->starRepo}";
```

Comment:
Not every repository is hosted on GitHub, maybe it would be better to give the full url to the repo? Alternatively do we read the `composer.json` we might infer the repository from there if it is set.

Reply:
Makes sense, I'll change this!